amazon-sqs/5.9/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,97 @@
+= Amazon SQS Connector 5.9 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
+== Long and Short Polling
+
+Since version 5.9, the default behavior of the Receive Messages source and the Read operation is to use long polling:
+
+* With long polling, SQS Connector returns the query results as soon as possible, if any messages are present.
++
+Otherwise, SQS Connector waits the specified amount of seconds before returning an empty response.
+* With short polling, SQS Connector always returns the query results as soon as possible, even if that means returning an empty response.
+
+Long polling can reduce the number of network requests without affecting the throughput.
+
+You can control polling behavior either globally or at the individual source and operation level.
+
+=== Control Polling Behavior Globally
+
+To control polling behavior globally, use the Default Wait Time field on the Advanced tab of the global element configuration screen for the Receive Messages source, Read operation, or both.
+
+image::amazon-sqs-config-wait-time.png[Default Wait Time Configuration]
+
+Set this field to an integer value between `1` and `20` (the default) to specify long polling as the default behavior. The integer specifies how long to wait, in time units specified in the Time unit field, before returning an empty response.
+
+Set this field to `0` to specify short polling as the default behavior.
+
+=== Control Polling behavior Locally
+
+To control polling behavior locally, use the Wait Time field on the Advanced tab of the Receive Messages source properties window, the Read operation properties window, or both.
+
+*Question for Reviewers asked in the Slack channel. Why are the polling fields missing for the Read operation in Studio? Only the Wait Time field is showing up there.*
```

Comment:
Could devs provide value details on what to set for the *Wait time* field? You can then write something similar to my previous comment of the *Default Wait Time* field.